docs/user/analytics.rst

```diff
@@ -1,5 +1,8 @@
-Traffic Analytics
-=================
+How To Use Traffic Analytics
```

Comment:
```suggestion
How to use traffic analytics
```

Question: Should we turn our features into Proper Nouns? In this case, that would make the headline be: "How to use Traffic Analytics".

Reply:
Nah, maybe not... simply looking at the amount of work to refactor all the texts makes this unrealistic :)